compat/src/util.js

```diff
@@ -10,6 +10,8 @@ export function assign(obj, props) {
 	return /** @type {O & P} */ (obj);
 }
 
+export const IS_NON_DIMENSIONAL = /^(-|f[lo].*[^se]$|g.{5,}[^ps]$|z|o[pr]|(W.{5})?[lL]i.*(t|mp)$|an|(bo|s).{4}Im|sca|m.{6}[ds]|ta|c.*[st]$|wido|ini)/;
```

Comment:
Is this the regex that was suggested a while back? I am wondering if the increased repetition will impact performance.

Reply:
It's the new one, while it is a bit slower it's more correct;

![image](https://user-images.githubusercontent.com/17125876/120685045-b4f91900-c49f-11eb-89f1-51a72335054c.png)

Reply:
imo it's perfectly fine as the `style` attribute isn't used that much in my experience. Most React apps I've worked on, only make use of that occasionally for certain areas.